public/src/reimbursement-policy.md

```diff
@@ -49,43 +49,57 @@ Speedcubing Canada’s Board of Directors reserves the right to modify this docu
 
 - Reimbursement requests are allowed to be made for equipment used for a Speedcubing Canada Supported Event.
 
-#### Type 1
+#### Type 1: timing equipment
```

Comment:
@kr-matthews interested in your feedback on the re-organization of this section. Thanks!

Reply:
It looks a lot clearer, thanks.
Is there a difference between reimbursement for equipment for a particular supported event vs for general equipment to be used at all supported events?

Reply:
For now, I would just name a recent or upcoming Supported Event to associate the cost with (I assume we're buying stuff to eventually use at an event).

In the future, I'd like to have a way to request equipment/reimbursement for general use as you described, but that's outside the scope of this round of edits.

Reply:
Including major expenses (like a laptop or printer) in a particular competition makes it hard to balance the budget, but having them separate and keeping all budgets (roughly) balanced doesn't seem sustainable mathematically.

Reply:
We are tracking general equipment purchases (for items procured directly by Speedcubing Canada) under a separate, non-competition budget.

Let me think about how we can better support "general equipment" in this policy...

Reply:
I added some flexibility for "general equipment" expenses: https://github.com/Speedcubing-Canada/docs-speedcubing-canada/pull/4/commits/feb8239cca939bdaea134b4c3f52ebdca2e130dd.

Still want to maintain some rigidity - let me know if you think this works.

Reply:
I think this is good, thanks. My only theoretical concern would be regions consistently running balanced budgets, but including lots of expenses under "general equipment" and therefore constantly losing money. I don't think it's a practical concern at this point though.